File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.java
import java.util.Arrays;
import java.util.stream.IntStream;

class Solution {
    public int solution(int[] sides) {
        int max = Arrays.stream(sides).max().getAsInt();
        int min = Arrays.stream(sides).min().getAsInt();
        int cnt = 0;
        
        long cnt1 = IntStream.range(max - min + 1, max + 1).count();
        long cnt2 = IntStream.range(max + 1, max + min).count();
        
        return (int)(cnt1 + cnt2);
    }
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.py
def solution(sides):
    max_num = max(sides)
    min_num = min(sides)
    cnt1 = len([cnt for cnt in range(max_num - min_num + 1, max_num + 1)])
    cnt2 = len([cnt for cnt in range(max_num + 1, max_num + min_num)])
    return cnt1 + cnt2